OSDOCS-11897_rn#release note IAM instance profile (#81949)
openshift/openshift-docs: release_notes/ocp-4-17-release-notes.adoc

[id="ocp-4-17-installation-and-update_{context}"]
=== Installation and update

==== Installing a cluster on {aws-full} using an existing IAM profile

> With this release, you can install {product-title} on {aws-short} by using an existing identity and access management (IAM) instance profile. You can specify either an existing IAM profile by using the `iamRole` parameter or you can specify a new IAM profile by using the new `iamProfile` parameter, but you cannot specify both. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].

Reviewer: I don't think we need to mention `iamRole` in the release note, as we already have a clear description in the [parameter table](https://github.com/openshift/openshift-docs/pull/81715)

```suggestion
With this release, you can install {product-title} on {aws-short} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
```

For more information on required permissions, see xref:../installing/installing_azure/installing-azure-account.adoc#minimum-required-permissions-ipi-azure_installing-azure-account[Required Azure permissions for installer-provisioned infrastructure].
====

> ==== Installing a cluster on {aws-full} using an existing IAM profile

Reviewer 2: ```suggestion
==== Installing a cluster on {aws-short} by using an existing IAM profile
```
I would suggest to consider using "AWS" in the heading and the full form in the paragraph. This also makes a heading smaller.

> With this release, you can install {product-title} on {aws-short} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].

Reviewer 2: ```suggestion
With this release, you can install {product-title} on {aws-first} by using an existing identity and access management (IAM) instance profile. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional {aws-short} configuration parameters].
```
